glean-core/rlb/src/private/ping.rs

```diff
@@ -2,13 +2,19 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+use inherent::inherent;
+
 /// A Glean ping.
 #[derive(Clone, Debug)]
 pub struct PingType {
     pub(crate) name: String,
     pub(crate) ping_type: glean_core::metrics::PingType,
 }
 
+// IMPORTANT:
+//
+// When changing this implementation, make sure all the operations are
+// also declared in the related trait in `../traits/`.
```

Comment:
I don't understand this comment, 
1. I couldn't find the traits folder you are pointing to here; 
2. Shouldn't this comment be on line 47 before `impl glean_core::traits::Ping for PingType`? 
3. Wouldn't Rust error anyways if you don't implement a trait function that doesn't include a default?

Reply:
Gah, you're right. This comment should be removed. Copy pasta.